packages/temen/src/toArray/index.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * `value`를 배열로 변환합니다.
+ *
+ * @param {*} value 배열로 변환할 값
+ * @returns {Array} 변환된 배열을 반환합니다.
+ * @example
+ *
+ * _.toArray({ 'a': 1, 'b': 2 });
+ * // => [1, 2]
+ *
+ * _.toArray('abc');
+ * // => ['a', 'b', 'c']
+ *
+ * _.toArray(1);
+ * // => []
+ *
+ * _.toArray(null);
+ * // => []
+ */
+
+import { isString } from '../is';
+import isObject from '../isObject';
+
+function toArray(value: any) {
+  if (isObject(value)) return Object.values(value);
+  if (isString(value)) return value.split('');
+
+  return [];
+}
```

Comment:
이거 이 타입만 봐서는 `toArray`가 Object와 String 타입만 처리한다는 것을 알기 어려울 것 같은데, 타입 오버로딩을 사용해서 더 명확하게 표현해주는 것은 어떨까유.

Reply:
https://github.com/Lubycon/lubycon-frontend-libraries/pull/149/commits/f97420681d592db630795942ccdf692e144e704f 오브젝트 타입 주기가 애매한데 이방법이 최선일까요?

Reply:
엇 그렇게 해도 결국 `toArray`는 `any[]`를 반환하게 되어서 타이핑이 조금 헐거울 것 같아유. 저는 

```suggestion
function toArray(value: boolean): [];
function toArray(value: number): [];
function toArray(value: string): Array<string>;
function toArray<T>(value: Record<string, T>): T[];
function toArray<T>(value: any) {
  if (isObject(value)) return Object.values<T>(value);
  if (isString(value)) return value.split('');

  return [];
}
```

이런 느낌을 생각했습니다. 어차피 `Date`, `Function` 이런 애들은 `isObject`에 걸릴테니 원시 자료형만 오버로딩해줘도 괜찮을 것 같아유

Reply:
아ㅏㅏㅏ확실히 요렇게 주는게 더 낫겠네요 👍

Reply:
https://github.com/Lubycon/lubycon-frontend-libraries/pull/149/commits/68b5294d3e8ca9279cd760c875e7e75dc66301e9

[[feat: reflect the code review]()](https://github.com/Lubycon/lubycon-frontend-libraries/pull/149/commits/6fa547fc9b682d6ac0f74113a3081ab69c6d72a1) 반영했숩니다 ~